### PR TITLE
SPCTR-2470-fix: color not visible in backend and fontStyles issue of content in inline-notice block

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description
Color and FontStyles issue of content in inline-notice block.
### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -199,6 +199,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Global - Excluded BR Tags from buttons' text.
 * Fix: Global - Cartflows templates throw an error in the editor's Spectra Page Settings.
 * Fix: Global - Fonts was not loading on 404 page when Spectra blocks used in Astra theme's header/footer.
+* Fix: Inline-Notice - Content color and FontStyle issues.
```
Comment (aapatre) [inline readme.txt:202]: Please add more detailed/specific bug description in the changelog.

**Review by aapatre — CHANGES_REQUESTED** [review]
Add more specific title and description to the PR.

File: readme.txt (hunk 1 above)
Comment (rajmohanreddy774) [reply readme.txt]: Added

**Review by aapatre — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -199,6 +199,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Global - Excluded BR Tags from buttons' text.
 * Fix: Global - Cartflows templates throw an error in the editor's Spectra Page Settings.
 * Fix: Global - Fonts was not loading on 404 page when Spectra blocks used in Astra theme's header/footer.
+* Fix: Inline-Notice - Content color not visible in backend and FontStyle not loading for Content.
```
Comment (RajkiranBagal) [inline README.md:202]:
```suggestion
* Fix: Inline Notice - Content color is not working in the backend.
* Fix: Inline Notice - Content typography is not working.
```

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rajmohanreddy774) [reply README.md:206]: Added as per suggestion

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**